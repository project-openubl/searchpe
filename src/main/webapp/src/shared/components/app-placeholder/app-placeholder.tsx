import React from "react";
import { Bullseye, Spinner } from "@patternfly/react-core";

export const AppPlaceholder: React.FC = () => {
  return (
    <Bullseye>
      <div className="pf-u-display-flex pf-u-flex-direction-column">
        <div>
          <Spinner />
        </div>
        <div className="pf-c-content">
          <h3>Loading...</h3>
        </div>
      </div>
    </Bullseye>
  );
};
