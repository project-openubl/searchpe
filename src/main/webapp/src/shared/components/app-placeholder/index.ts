export { AppPlaceholder } from "./app-placeholder";
