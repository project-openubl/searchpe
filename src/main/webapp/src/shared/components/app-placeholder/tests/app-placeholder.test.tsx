import React from "react";
import { shallow } from "enzyme";
import { AppPlaceholder } from "../app-placeholder";

describe("AppPlaceholder", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<AppPlaceholder />);
    expect(wrapper).toMatchSnapshot();
  });
});
