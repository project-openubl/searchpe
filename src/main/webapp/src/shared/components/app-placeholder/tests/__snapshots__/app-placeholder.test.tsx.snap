// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPlaceholder Renders without crashing 1`] = `
<Bullseye>
  <div
    className="pf-u-display-flex pf-u-flex-direction-column"
  >
    <div>
      <Spinner />
    </div>
    <div
      className="pf-c-content"
    >
      <h3>
        Loading...
      </h3>
    </div>
  </div>
</Bullseye>
`;
